### DISCUSSION TITLE
Lowering the "floor" of the bin internals?

### DISCUSSION THREAD

**Discussion by Jeppedy** [Le0Michine/FusionGridfinityGenerator#129] [Q&A]
Better to explain my need:

- I'm making bespoke tool holders.
- I generate the bin with a solid interior
- I bring in the model of my tool, combine/cut from the bin
- That works great, but now half of the part sticks ABOVE the bin, with the other half recessed below the bin top.

What I'd like to do is LOWER the bin's "floor" so that the top of my tool is BELOW the lip of the bin (or stacking lip, if chosen)
  I've tried making it fully shelled, then adding a layer to embed the tool body into, but it makes a "bottom layer" that I don't need.
  I've tried making it a solid bin, then adding a cut to it (not really ideal, but works)
Can't I just lower the height of the bin internals?  (Tried to find the extrude step to change that but was unsuccessful)
I'm a bit new to Fusion and could use a nudge here!

**Comment by griffithska** [comment]
I'm brand new here so maybe I'm off base, but I hope this helps. I've been playing around with the settings and reading posts to wrap my head around all the options. I think you can accomplish what you're trying to do by setting the Bin Compartments to 1x1 and then set Grid Type to Custom. Add 1 custom compartment, X = 0, Y = 1, Width = 1, Length = 1 and then set the depth to where you want your "floor" to be. That should give you a bin that is somewhere between Hollow and Solid so you can recess your cutout.

![image](https://github.com/user-attachments/assets/4d218466-b887-4a33-85f0-caa728691674)
